examples/palettetest/Makefile

```diff
@@ -0,0 +1,90 @@
+# Please see the README for setting up a valid build environment.
+
+# Commands
+MKDIR 		:= mkdir
```

Comment:
Not to split hairs, but I don't see this used anywhere?

Reply:
Okay well I am exactly splitting hairs lmao, sorry. Anyway, am I missing where this is used?

Reply:
Lmao yeah I just left this in by accident; the palette test spun off a different project and I must've copy-pasted that over by accident. I'll remove it in a quick revision. Apologies!